cypress/integration/Autocomplete.spec.ts

```diff
@@ -0,0 +1,348 @@
+import * as h from '../helpers';
```

Comment:
This whole file is copied from https://github.com/Workday/canvas-kit/blob/9304d136509aed73b0f272dc62d4609d8b91be36/cypress/integration/ComboboxLabs.spec.ts

Some lines were updated to point to the correct story and some tests are skipped because grouping isn't supported yet.